File: 30-Days-Of-Code/Day 2/Operators.cpp
#include <iostream>
#include <math.h> /*since we have used the round function.*/

using namespace std;

int main()
{
    double cost, total=0;
    int tip, tax;

    cin>>cost>>tip>>tax;

    total = cost + tip*cost/100 + tax*cost/100; /*calculation*/
   
    cout<<"The total meal cost is "<<round(total)<<" dollars."; /*to round off the answer.*/

return 0;
}

File: 30-Days-Of-Code/Day 2/Operators.js
 /* default conditions are already given in editor of javascript(node.js) language */
 
  let tip = mealCost * tipPercent/100;                //let is used to declare variables and is latest syntax 
  let tax = mealCost * taxPercent/100;
  let cost = Math.round(mealCost + tax + tip);
  console.log("The total meal cost is "+cost+" dollars.");   //to print the output
